virt-who.8

```diff
@@ -36,12 +42,17 @@ In this mode virt-who just sends the host to guest association to the server onc
 2. interval mode
 # virt-who -i INTERVAL
 
-This is default mode. virt-who will listen to change events (if available) or do a polling with given interval, and will send the host to guest association when it changes. The default polling interval is 3600 seconds and can be changed using "-i INTERVAL" (in seconds).
+This is the default mode. virt-who will listen to change events (if available) or do a polling with given interval, and will send the host to guest association when it changes. The default polling interval is 3600 seconds and can be changed using "-i INTERVAL" (in seconds).
 .TP
 3. print mode
 # virt-who -p
 
 This mode is similar to oneshot mode but the host to guest association is not send to server, but printed to standard output instead.
+.TP
+3. status mode
+# virt-who -s
+
+This mode is for configuration diagnosis. The host to guest association is not compiled and is not sent to the server.
```

Comment:
I'd suggest we tell the users a bit more about what is actually happening here.
As an example, how do you use this to diagnose issues with configurations?
What should a user expect to see?